### PR TITLE
#71252 disable Atan2PiTest on Android x86

### PR DESCRIPTION
**Pull request by mkhamoyan**
Disable Atan2PiTest on Android x86 #71252

### CODE REVIEW

**Review by steveisok — APPROVED** [review]

**Review by stephentoub — APPROVED** [review]

File: src/libraries/System.Runtime/tests/System/DoubleTests.cs [hunk 1]
```diff
@@ -1486,7 +1486,7 @@ public static void Atan2PiTest(double y, double x, double expectedResult, double
             AssertExtensions.Equal(+expectedResult, double.Atan2Pi(+y, +x), allowedVariance);
         }
 
-        [Theory]
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotAndroid), nameof(PlatformDetection.IsNotX86Process))]   // disabled on Android x86, see https://github.com/dotnet/runtime/issues/71252
```
[the remark this answers is not in the document]
Comment (mkhamoyan) [reply src/libraries/System.Runtime/tests/System/DoubleTests.cs:1489]: From my understanding it works like PlatformDetection.IsNotAndroid && PlatformDetection.IsNotX86Process

[the remark this answers is not in the document]
Comment (stephentoub) [reply src/libraries/System.Runtime/tests/System/DoubleTests.cs:1489]: Right, so if it is x86, it fails the right clause, and that whole expression is false.

[the remark this answers is not in the document]
Comment (stephentoub) [reply src/libraries/System.Runtime/tests/System/DoubleTests.cs]: You just want `!(IsAndroid && Isx86)`

**Review by stephentoub — APPROVED** [review]

**Comment by akoeplinger** [comment]
Thanks. There's a good chance we'll be able to reenable those tests once https://github.com/dotnet/runtime/pull/65723 lands.

**Pull request merged**